<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QDialog" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/chat/image/icon/icon_title1.png</normaloff>:/chat/image/icon/icon_title1.png</iconset>
  </property>
  <widget class="QLabel" name="WatermelonChat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/title.png);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>中国式手写风</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/background_peopleleft.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="fontSizeComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>520</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    /*min-width: 9em;   组合框的最小宽度*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}
QComboBox::down-arrow {
image: url(:/chat/image/icon/icon_menu.png);
}
QComboBox::drop-down:hover { background-color: rgb(50, 50, 50,255);}</string>
   </property>
   <property name="currentText">
    <string>9</string>
   </property>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="textUnderline">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_underline.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="textButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>701</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-left-radius:20px; 
border-bottom-right-radius:20px; 
background-color: rgb(255, 255, 255, 230);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
    /*min-width: 9em;   组合框的最小宽度*/
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; 
    border-top-right-radius: 3px; 
    border-bottom-right-radius: 3px;
}
QComboBox::down-arrow {
	image: url(:/chat/image/icon/icon_menu.png);
}
QComboBox::drop-down:hover { background-color: rgb(50, 50, 50,255);}</string>
   </property>
   <property name="currentText">
    <string>宋体</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_pic.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SendButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>740</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>103</width>
     <height>41</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(172, 178, 148,200); border-radius: 3px; color: rgb(255, 255, 255); }
 QPushButton:hover { background-color: rgb(172, 178, 148,255);}
</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditWrite">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>461</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-left-radius:0px; 
border-top-right-radius:0px;
background-color: rgb(255, 255, 255,230);
border-top:2px solid #e1e0e0;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditRead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>461</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 230);
border-top-left-radius:20px; 
border-top-right-radius:20px;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_download.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textcolor">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_color.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="clear">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255);border-image: url(:/chat/image/icon/icon_del.png); }
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="emojiButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_emoji.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>730</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_novoice.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textitalic">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_italic.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="sendfile">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>730</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(50, 50, 50,0); border-radius: 6px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_file.png);}
QPushButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="textbold">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>520</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); border-image: url(:/chat/image/icon/icon_bold.png);}
QToolButton:hover { background-color:rgba(172, 178, 148, 100)}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gameButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>750</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/chat/image/icon/icon_game.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="hideButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>最小化</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); }
QToolButton:hover { background-color: rgb(172, 178, 148);}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/chat/image/icon/icon_hide.png</normaloff>:/chat/image/icon/icon_hide.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton { background-color: rgb(50, 50, 50,0); border-radius: 3px; color: rgb(255, 255, 255); }
QToolButton:hover { background-color: rgb(172, 178, 148);}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/chat/image/icon/icon_close.png</normaloff>:/chat/image/icon/icon_close.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="webButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>750</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton { background-color: rgb(172, 178, 148,100); border-radius: 3px; color: rgb(255, 255, 255); }
 QPushButton:hover { background-color: rgb(172, 178, 148,150);}
</string>
   </property>
   <property name="text">
    <string>follow我们的GitHub</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>291</width>
     <height>611</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>801</width>
     <height>801</height>
    </rect>
   </property>
   <widget class="QWidget" name="page"/>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <zorder>stackedWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>textEditRead</zorder>
  <zorder>WatermelonChat</zorder>
  <zorder>label</zorder>
  <zorder>fontSizeComboBox</zorder>
  <zorder>textUnderline</zorder>
  <zorder>textButton</zorder>
  <zorder>fontComboBox</zorder>
  <zorder>imageButton</zorder>
  <zorder>SendButton</zorder>
  <zorder>textEditWrite</zorder>
  <zorder>save</zorder>
  <zorder>textcolor</zorder>
  <zorder>clear</zorder>
  <zorder>emojiButton</zorder>
  <zorder>voiceButton</zorder>
  <zorder>textitalic</zorder>
  <zorder>sendfile</zorder>
  <zorder>textbold</zorder>
  <zorder>gameButton</zorder>
  <zorder>hideButton</zorder>
  <zorder>closeButton</zorder>
  <zorder>webButton</zorder>
  <zorder>widget</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
